src/types/commands.ts

```diff
@@ -118,6 +118,7 @@ export const invalidateEvaluationCommands = new Set<CommandTypes>([
   "UNDO",
   "REDO",
   "ADD_MERGE",
+  "REMOVE_MERGE",
```

Comment:
we should take this opportunity to update `invalidateEvaluationCommands` with the command `DUPLICATE_SHEET` as well. (**in a different commit**) 

This would allow to remove the useless lines here https://github.com/odoo/o-spreadsheet/blob/master/src/plugins/ui_core_views/cell_evaluation/evaluation_plugin.ts#L192 

But should also come with some tests for DUPLICATE_SHEET.

Reply:
I think adding a test is unnecessary in this case. If the `DUPLICATE_SHEET` command is removed from EvaluationPlugin without being added to `invalidateEvaluationCommands`, numerous existing test cases will fail, and the UI will break.

Reply:
lk